build.sbt

```diff
@@ -17,25 +17,36 @@ lazy val root = (project in file(".")).
   aggregate(saulCore, saulExamples)
   .enablePlugins(AutomateHeaderPlugin)
 
+lazy val docSettings = Seq(
+  autoAPIMappings := true,
+  apiURL := Some(url("http://cogcomp.cs.illinois.edu/software/doc/saul/")),
+  scalacOptions in Test ++= Seq("-Yrangepos")
+)
+
 lazy val commonSettings = Seq(
-  organization := "edu.illinois.cs.cogcomp",
+  organization := ccgGroupId,
   name := "saul-project",
-  version := "0.4",
+  version := "0.5",
   resolvers ++= Seq(
     Resolver.mavenLocal,
     "CogcompSoftware" at "http://cogcomp.cs.illinois.edu/m2repo/"
   ),
   javaOptions ++= List("-Xmx11g"),
   libraryDependencies ++= Seq(
-    "edu.illinois.cs.cogcomp" % "LBJava" % "1.2.20" withSources,
-    "edu.illinois.cs.cogcomp" % "illinois-core-utilities" % cogcompNLPVersion withSources,
+    ccgGroupId % "LBJava" % "1.2.20" withSources,
+    ccgGroupId % "illinois-core-utilities" % cogcompNLPVersion withSources,
     "com.gurobi" % "gurobi" % "6.0",
     "org.apache.commons" % "commons-math3" % "3.0",
     "org.scalatest" % "scalatest_2.11" % "2.2.4",
     "ch.qos.logback" % "logback-classic" % "1.1.7"
   ),
   fork := true,
-  publishTo := Some(Resolver.sftp("CogcompSoftwareRepo", "bilbo.cs.illinois.edu", "/mounts/bilbo/disks/0/www/cogcomp/html/m2repo/")),
+  publishTo := Some(
+    Resolver.ssh(
+      "CogcompSoftwareRepo", "bilbo.cs.illinois.edu",
+      "/mounts/bilbo/disks/0/www/cogcomp/html/m2repo/").
+      as ("khashab2", new java.io.File(Path.userHome.absolutePath + "/.ssh/key"))
+  ),
```

Comment:
That's a good point. I know it is idea. In fast I tried that but did not work:
http://stackoverflow.com/questions/39261243/sbt-credentials-java-lang-illegalargumentexception-user-must-not-be-null
This is really the limitation of ivy.